src/sds_data_model/metadata.py

```diff
@@ -169,17 +173,21 @@ def _get_value(
     target_elements = _get_target_elements(
         root_element,
         xpath=xpath,
-        namespaces=namespaces,
-    )
-    target_element: str = target_elements[0].strip()
+        namespaces=namespaces,)
+
+    if not target_elements:
+        target_element = None
+    else:
+        target_element: str = target_elements[0].strip()
```

Comment:
`mypy` is complaining because your annotating `target_element` on line 181 after you've first defined it on line 179. As the return type of this function is now `Optional[str]`, you shouldn't need to assert that `target_element` is string here anymore. 
```suggestion
    if not target_elements:
        target_element = None
    else:
        target_element = target_elements[0].strip()
```